### PR TITLE
Adding collaborator triggers fieri

### LINKED ISSUE #1378: Adding a new collaborator needs to triger quality re-eval
Collaborators isn't version specific. If someone gets a bad score and they go about fixing it by adding additional collaborators they shouldn't have to upload a new cookbook. We should re-eval the bad score when they add more collaborators.

### PR DESCRIPTION
**Pull request by loganmeetsworld**
Fixes #1378 pending testing it out in a Terraform environment.

### CODE REVIEW

File: src/supermarket/app/controllers/collaborators_controller.rb [hunk 1]
```diff
@@ -40,6 +40,10 @@ def create
 
       add_group_members_as_collaborators(resource, collaborator_params[:group_ids]) if collaborator_params[:group_ids].present?
 
+      if collaborator_params[:resourceable_type] == 'Cookbook'
```
Comment (nellshamrell) [inline src/supermarket/app/controllers/collaborators_controller.rb]: Change to testing whether class of resource is a Cookbook?

**Comment by loganmeetsworld** [comment]
Tested in Terraform and it works.

**Comment by nellshamrell** [comment]
👍 Merge at will!

File: src/supermarket/app/controllers/collaborators_controller.rb (hunk 1 above)
Comment (loganmeetsworld) [inline src/supermarket/app/controllers/collaborators_controller.rb]: @nellshamrell @robbkidd can I get a 👍  on not changing this or are there opinions that I should change it to `resource.class == 'Cookbook'`? Definitely willing to do either way.

Comment (nellshamrell) [inline src/supermarket/app/controllers/collaborators_controller.rb]: Give it a 20 min spike and see if changing it breaks any tests?

Comment (loganmeetsworld) [inline src/supermarket/app/controllers/collaborators_controller.rb]: It worked and reads a little better so we'll go with this.

**Pull request merged**